org.devmiyax.yabasanshiro.appdata.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2019 Nick Richards -->
+<component type="console-application">
+  <id>org.devmiyax.yabasanshiro</id>
```

Comment:
`devmiyax.org` doesn't exist.

app-id must refer to an existing domain under the control of the developer.